integration_tests/models/schema_tests/schema.yml

```diff
@@ -18,7 +18,14 @@ models:
               regex: "[A-Z]"
               flags: i
               config:
-                enabled: "{{ target.type in ['postgres', 'snowflake', 'redshift' ] }}"
+                enabled: "{{ target.type in ['postgres', 'snowflake', 'redshift', 'bigquery' ] }}"
```

Comment:
This covers all of our support platform, I'm wondering if we should just remove the enabled config for this test then?

Reply:
I've now removed all occurrences of that enabled config